### ISSUE TITLE
Xenial Jobs Failing with New Ubuntu Docker Image

### ISSUE THREAD

**Issue by evenator** [ros-infrastructure/ros_buildfarm#414]
The `latest` tag of the Ubuntu Xenial image at https://hub.docker.com/r/library/ubuntu/ has just been updated to a new image. On our internal build farm, we're seeing all Xenial source jobs failing in Step 4 of the `Build Dockerfile - generate sourcedeb` stage with this failure:

```
Step 4 : RUN locale-gen en_US.UTF-8
 ---> Running in b44c39e2a893
[91m/bin/sh: 1: locale-gen: not found
[0mRemoving intermediate container b44c39e2a893
The command '/bin/sh -c locale-gen en_US.UTF-8' returned a non-zero code: 127
Build step 'Execute shell' marked build as failure
```

It would seem that the new image doesn't have `locale-gen` pre-installed. As far as I can tell, this hasn't hit the main ROS buildfarm at https://build.ros.org (perhaps due to caching).

With the previous `xenial`, `locale-gen` runs ok:

```bash
$ sudo docker run -it --rm ubuntu:xenial-20170214
root@78055af4ac09:/# locale-gen
Generating locales (this might take a while)...
Generation complete.
```

But with the current `xenial:latest` image, it fails:

```bash
$ sudo docker run -it --rm ubuntu:xenial
root@1eaa25e24244:/# locale-gen
bash: locale-gen: command not found
```

**Comment by mikaelarguedas** [comment]
thanks for reporting this. Docker images on the buildfarm are invalidated daily, that would explain why it hasn't hit the main ROS buildfarm yet.

**Comment by mikaelarguedas** [comment]
Should be fixed by #415 .

Thanks again @evenator for the heads up!
